noir-projects/noir-protocol-circuits/src/crates/rollup-lib/src/base/base_rollup_inputs.nr

```diff
@@ -256,19 +256,19 @@ impl BaseRollupInputs {
     fn components_compute_kernel_calldata_hash(combined: CombinedAccumulatedData) -> [Field; NUM_FIELDS_PER_SHA256] {
         // Compute calldata hashes
         // Consist of
-        // MAX_NEW_NOTE_HASHES_PER_TX fields for commitments
+        // MAX_NEW_NOTE_HASHES_PER_TX fields for note hashes
         // MAX_NEW_NULLIFIERS_PER_TX fields for nullifiers
-        // 32 public data update requests -> 64 fields
         // 2 l2 -> l1 messages -> 2 fields
+        // 32 public data update requests -> 64 fields
         // 1 contract deployments -> 3 fields
         // 1 encrypted logs hashes -> 2 fields --> 2 sha256 hashes --> 64 bytes
```

Comment:
Something here looks off. `2 fields -> 2 sha256 hashes` that don't seem correct. Those are just the 32 bytes no? The code is looking a bit funky but all the way at the end it is only loading in 32 bytes for each.

Reply:
Fixed in https://github.com/AztecProtocol/aztec-packages/pull/4726/commits/b1baf724a03c339ef1b120089ef2722b42168102